contents/blog/seed-grow-scale-devrel.mdx

```diff
@@ -0,0 +1,217 @@
+- **Community**: Creating and managing programs for customers, partners, and third-party communities. At some companies, this team can also support and enable community within the company.
+- **Integrations & Partnerships**: Integrations with third-party technologies used by developers, direct integrations with key customers, or helping build an ecosystem with partners such as those seen with Slack apps, Salesforce apps etc.
+
+In a **Multi-Functional** team set up where a team consists of Product Managers, Engineers, DevRel roles, and Marketing roles, the specialization may change from sprint to sprint as a product moves through the product life-cycle; from inception/proof of concept through to a go-to-market phase. _Occasionally_ changing specialization can be a good setup for a generalist developer advocate, allowing them to apply their broad range of skills but still have time to focus in stages as product development progresses.
+
+For example, the DevRel role's focus may shift:
+
+- **Proof of Concept**: at this stage, the person in the DevRel role may be "customer zero" (see [the developer advocate as customer zero](https://developerrelations.com/dev-rel/the-developer-advocate-as-customer-zero)), providing feedback and acting as a customer.
+- **Development**: Continuing to act as "customer zero" as well as preparing the assets and structure for the beta (or other early access) program.
+- **Beta**: Executing on the beta plan, using this stage to raise awareness and excitement with developers, enabling developers with the product, and getting product feedback.
+- **Go-to-market**: Ramping up activities focused on awareness, acquisition, and activation.
+
+**Regional** teams are are often multi-functional but, as you'd expect, focus on a specific geographic location. You'll most likely see this as the number of DevRel roles at a company gets to a point where it's possible to justify having North American, EMEA, and APAC regional teams.
+
+It's always important to be collaborative. But there's also a lot of value in not having to collaborate on absolutely everything to get something done. Therefore, a team should be **empowered** as much as possible to do everything they need to get stuff done; equipment, software, skills, authority, budget, and more. If you're leading a DevRel team, this most definitely means support. Your role as a leader of a DevRel team - and any empowered team - is to support and enable them, not tell them what to do.
+
+### Work with customers, partners, and community
+
+When scaling, it's essential to think beyond simply adding headcount. That, in itself, isn't scaling. You're only really scaling a team if further investment increases output more than the increase in input. For example, if a Developer Education team of five people produces five pieces of content a week (one piece per person), adding another person should result in the team producing ten pieces of content a week.
```

Comment:
Why does 5 people = 5 pieces of content but 6 people = 10?

Reply:
I'm attempting to provide an example of what scaling might mean, where adding one person would just result in just one more piece of content.

Reply:
Clarified in 29730366c7f3fcbd6cba445381cebae4d53e89ae